parsers/US_CA.py

```diff
@@ -34,37 +34,51 @@ def fetch_production(country_code='US-CA', session=None):
       'source': 'mysource.com'
     }
     """
-    # caiso.com provides daily data until the day before today
-    # get a clean date at the beginning of yesterday
-    yesterday = arrow.utcnow().to('US/Pacific').shift(days=-1).replace(hour = 0, minute = 0, second = 0, microsecond = 0)
-    url = 'http://content.caiso.com/green/renewrpt/' + yesterday.format('YYYYMMDD') +'_DailyRenewablesWatch.txt'
-
-    renewableResources = pandas.read_table(url, sep='\t\t', skiprows=2, header=None, names=['Hour', 'GEOTHERMAL','BIOMASS', 'BIOGAS', 'SMALL HYDRO','WIND TOTAL', 'SOLAR PV', 'SOLAR THERMAL'], skipfooter=27, skipinitialspace=True, engine='python')
-    otherResources = pandas.read_table(url, sep='\t\t', skiprows=30, header=None, names=['Hour', 'RENEWABLES', 'NUCLEAR', 'THERMAL', 'IMPORTS', 'HYDRO'], skipinitialspace=True, engine='python')
-
-    dailyData = []
+    url = 'http://www.caiso.com/outlook/SP/fuelsource.csv'
+    csv = pandas.read_csv(url)
+    latest_index = len(csv)-1
```

Comment:
are you able to return an array of items instead of just the latest one? Then the parser will be more robust to downtimes.

Reply:
I could but that makes the issue stated below more problematic.

Reply:
If you can, please do (it will be more robust).
Else, we can do later.